### ISSUE TITLE
[oxygen] salt-cloud: rackspace skipped tests

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#652]
The rackspace salt-cloud tests are being skipped:

```
15:05:45    -> integration.cloud.providers.test_rackspace.RackspaceTest.test_instance                                           ->  Configuration file for rackspace was not foundConnection to <ipaddresss> closed.
```

**Comment by rallytime** [comment]
@Ch3LL What salt-cloud driver is being used here? I think that's intended from the changes @gtmanfred made for the openstack driver. He has a plan for getting the new openstack driver tested (using kitchen-salt, iirc).

**Comment by gtmanfred** [comment]
hrm, that one shouldn't be getting skipped though.

**Comment by Ch3LL** [comment]
Yeah i'm not sure we will just need someone to dive more into it to see why its not picking up. Could just be an auth problem.

**Comment by gtmanfred** [comment]
Oh yeah, we probably need to switch the settings over to be the new
openstack driver, instead of the old one possibly?

**Comment by gtmanfred** [comment]
Once the above to prs are in, we will just need to update the providers and profiles for the cloud tests, and this should be ready.

**Comment by Ch3LL** [comment]
I'll test this today or tomorrow

**Comment by Ch3LL** [comment]
Once this is merged: https://github.com/saltstack/salt-jenkins/pull/667  tests should start passing since the cloud tests use the master branch.

**Comment by gtmanfred** [comment]
:+1:

**Comment by Ch3LL** [comment]
pending this merge: https://github.com/saltstack/salt/pull/45306

**Comment by Ch3LL** [comment]
tests are not being skipped now although they are failing now. Will create another issue to track the failure and fix.

**Comment by Ch3LL** [comment]
here's the issue for reference: https://github.com/saltstack/salt-jenkins/issues/682
